src/core/AutomationPattern.cpp

```diff
@@ -538,6 +538,11 @@ void AutomationPattern::saveSettings( QDomDocument & _doc, QDomElement & _this )
 	_this.setAttribute( "prog", QString::number( progressionType() ) );
 	_this.setAttribute( "tens", QString::number( getTension() ) );
 	_this.setAttribute( "mute", QString::number( isMuted() ) );
+	
+	if( hasColor() && usesCustomClipColor() )
```

Comment:
`usesCustomClipColor()` implies `hasColor()`, so this can be simplified to just `usesCustomClipColor()`.

Reply:
Yepp that was a blunder.... fixed it xD